contracts/ConsensusUtils.sol

```diff
@@ -571,4 +576,22 @@ contract ConsensusUtils is EternalStorage, ValidatorSet {
   function _setValidatorFee(address _validator, uint256 _amount) internal {
     uintStorage[keccak256(abi.encodePacked("validatorFee", _validator))] = _amount;
   }
+
+  function unboundingAmount() public view returns(uint256) {
```

Comment:
if queue is long this will fail if used inside  a smart contract.
we can keep this, but add another view helper   canWithdraw() which simply returns true if first in queue can be withdrawn.
so some contract can easily call unbound() then check if still canWithdraw() and call unbound again

I still dont think it is necessary to limit the queue length, as it is per account, that account owner can only attack itself.
for example in our use case, we can simply limit users to withdraw at least 10% of their funds and at least 1 fuse, so they cant create a huge queue withdrawing small amounts